jwql/instrument_monitors/miri_monitors/data_trending/15min_to_db.py

```diff
@@ -0,0 +1,141 @@
+import statistics
+import os
+import jwql.instrument_monitors.miri_monitors.data_trending.utils.mnemonics as mn
+import jwql.instrument_monitors.miri_monitors.data_trending.utils.sql_interface as sql
+import jwql.instrument_monitors.miri_monitors.data_trending.utils.csv_to_AstropyTable as apt
+from jwql.utils.utils import get_config, filename_parser
+
+from jwql.instrument_monitors.miri_monitors.data_trending.utils.process_data import once_a_day_routine
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+#point to the directory where your files are located!
+directory = '/home/daniel/STScI/trainigData/set_15_min/'
+
+#here some some files contain the same data but they are all incomplete
+#in order to generate a full database we have to import all of them
+filenames = [

```

Comment:
Could that list be generated on the fly using `glob.glob(os.path.join(directory, 'imir_*.CSV'))` ?

Reply:
This would have saved a lot of time! thanks :)